widget/embedded/src/hooks/useStatefulConnect/useStatefulConnect.state.ts

```diff
@@ -34,11 +34,14 @@ export function reducer(state: State, action: Actions): State {
     case 'reset':
       return initState;
     case 'resetDerivation':
-      return {
-        ...state,
-        derivationPath: null,
-        status: 'namespace',
-      };
+      if (state.namespace) {
```

Comment:
Some wallets may haven't namespace (like trezor) yet, in this case we should go to `init` mode, instead of `namespace` mode.